programming/arduino/sr_firmware.md

```diff
@@ -57,10 +57,13 @@ The possible modes for a pin are:
 `INPUT_PULLUP`
 :   set the pin to input mode with a [pull-up resistor](#pull-up-resistors)
 
+These are available from `sr.robot3`, or the `GPIOPinMode` enum.
```

Comment:
The enum is an implementation detail.
```suggestion
These are available from `sr.robot3`.
```

Reply:
I'm not sure I agree about forgetting it. I'm sure there are cases where knowing that there's a backing enum is useful. Or even that the type of the value is an enum member (so `.value` exists, etc).